bonsai/ethereum/Cargo.toml

```diff
@@ -13,7 +13,7 @@ anyhow = "1.0"
 [dependencies]
 ethers = { version = "2.0", features = ["rustls", "ws"] }
 hex = { version = "0.4", optional = true }
-risc0-zkvm = { workspace = true, default-features = false, optional = true }
+risc0-zkvm = { workspace = true, features = ["prove"], optional = true }
```

Comment:
This feature is only needed for a utility binary to print out the control_id formatted for updating the verifier contract. I think we can drop the binary and just have a test for that. In this case we could also catch when we brake the control_id compatibility/Ethereum verifier contract